calcom/cal.com
# refactor: change title

## What does this PR do?



- Fixes #XXXX (GitHub issue number)
- Fixes CAL-XXXX (Linear issue number - should be visible at the bottom of the GitHub issue description)

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

 - Enable instant meeting in org team event and try to book a meeting to see the modal
